src/fragments/start/getting-started/vue/data-model.mdx

```diff
@@ -192,44 +192,43 @@ To display the data, update `App.vue` to list all the items in the database by i
 </template>
 
 <script>
-import { API } from 'aws-amplify';
-import { createTodo } from './graphql/mutations';
-import { listTodos } from './graphql/queries';
-
-export default {
-  name: 'App',
-  async created() {
-    this.getTodos();
-    this.subscribe();
```

Comment:
This was the only real change in this file. This was calling subscribe before it was added.